notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   "source": [
+    "    Note that it is critical for some use cases that the above should yield spectral regions that are correct *even to the sub-pixel*.  That is, if the user does a region selection that ends between two pixels, that should be reflected in the SpectralRegion - this means using subset *states* in glue instead of the subset itself.\n",
+    "\n",
+    "    These selections can now be used with standard specutils analysis functions in a notebook setting (these are [ALREADY IMPLEMENTED]):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specutils.analysis.line_flux(specviz.current_spectrum, specviz.get_selected_region())\n",
+    "\n",
+    "specutils.analysis.equivalent_width(specviz.current_spectrum, regions=specviz.get_selected_region())\n",
+    "\n",
+    "specutils.analysis.gaussian_fwhm(specviz.current_spectrum, specviz.get_selected_region())\n",
+    "\n",
+    "specutils.analysis.snr(specviz.current_spectrum, specviz.get_selected_region())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    In addition to region selection, there is value in having a way to extract a specific point in the displayed spectrum into the notebook. (Some examples are shown below) - while there are many potential solutions to this, the simplest is a stateful \"get location\" method.  E.g.,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spectral_coordinate, flux_coordinate = specviz.get_spectrum_location()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    This method would block, cause the cursor in the viz tool to change to some indicator cursor, and then un-block as soon as the user clicks somewhere in the viewer window, reporting out the spot the click."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the Desktop interface, relevant analysis functions like the above should be exposed as information plugins - when run this plugin would simply show the result of these functions as run in the latest `get_spectral_region` subset exactly as shown above but auto-update when the user creates a new/updates the current subset.\n",
+    "\n",
+    "    One additional related piece of functionality that should be added to the above plugins: provide a way to replicate the `e` key feature below.  That is, provide a quick way for the user to not just specify the spectral region, but also mark two points to do automatic continuum-specification.  This defines a simple linear continuum, which the plugin could subtract from and divide from the spectrum before doing the calculation desired.  How exactly the user specifies the two points is TBD, but perhaps could be a glue box-selection, or a special two-click stateful operation particular to the plugin."
+   ]
+  },
```

Comment:
Should allow multiple continuum fit windows, e.g. to either side of a single line or multiple lines, or across the spectrum.

Reply:
Yes, good- I was trying to highlight that the "quick and easy" "e" based workflow should be exposed, but I will clarify that more complicated fitting is *also* necessary.